src/configs/shared.ts

```diff
@@ -24,7 +24,7 @@ export const DEFAULT_TIME_RANGE_FOR_MAX_ISSUE_ENABLED = true;
 export const DEFAULT_FOLLOWUP_TIME = "4 days"; // 4 days
 export const DEFAULT_DISQUALIFY_TIME = "7 days"; // 7 days
 
-export const DEFAULT_CHAIN_ID = 1; // ethereum
+export const DEFAULT_NETWORK_ID = 1; // ethereum
```

Comment:
```suggestion
export const DEFAULT_EVM_NETWORK_ID = 1; // ethereum
```

Might be more clear what type of network we're talking about!